students/13th/jeehoonkim/westagram/westagram/urls.py

```diff
@@ -0,0 +1,22 @@
+"""westagram URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
+from django.urls import path, include
+
+urlpatterns = [
+    path('user/', include('user.urls')),
+    path('posting/', include('posting.urls')),
```

Comment:
```suggestion
    path('user', include('user.urls')),
    path('posting', include('posting.urls')),
```
위코드 규칙을 하나 더 알려드리자면, 최상위 url에는 슬래시를 붙이지 않고, 그 다음에 오는 app url의 앞에 슬래시를 붙이는 것을 추천드립니다 :)

Reply:
넵 수정했습니다🙃